### PR TITLE
Fix missing StackOverflow badge

### PR DESCRIPTION
**Pull request by jason-fox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![FIWARE Core Context Management](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/core.svg)](https://www.fiware.org/developers/catalogue/)
 [![License badge](https://img.shields.io/github/license/telefonicaid/fiware-orion.svg)](https://opensource.org/licenses/AGPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/fiware/orion.svg)](https://hub.docker.com/r/fiware/orion/)
-[![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware-orion)
+[![Support badge](https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow)](http://stackoverflow.com/questions/tagged/fiware-orion)
```
Comment (fgalan) [inline README.md:9]: I have tried to access directly to the URL https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow and I get:

imagen

It seems it is not working...

Comment (jason-fox) [reply README.md:9]: That is weird

```
![](https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow)
```

gives  for me.

Comment (fgalan) [reply README.md:9]: Now it's also working for me. Maybe it has been some kind of temporal glitch in img.shields.io

NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**